src/snowflake/snowpark/modin/plugin/extensions/pd_overrides.py

```diff
@@ -369,7 +370,7 @@ def read_json(
     path_or_buf: FilePath,
     *,
     orient: Optional[str] = None,
-    typ: Optional[Literal["frame", "series"]] = None,
+    typ: Optional[Literal["frame", "series"]] = "frame",
```

Comment:
can we add todo to track those changes?

Reply:
Changing this argument should be fine; this just determines whether the output of `read_json` is a DF or Series, and our default behavior is to produce a DF anyway. I added a comment at the top of `snowflake/snowpark/modin/pandas/io.py` with a TODO to remove those frontend stubs: SNOW-1650875